pkg/shares/shares_test.go

```diff
@@ -275,12 +274,12 @@ func TestFuzz_Merge(t *testing.T) {
 }
 
 // generateRandomBlockData returns randomly generated block data for testing purposes
-func generateRandomBlockData(txCount, evdCount, msgCount, maxSize int) coretypes.Data {
-	var out coretypes.Data
-	out.Txs = generateRandomlySizedCompactShares(txCount, maxSize)
-	out.Evidence = generateIdenticalEvidence(evdCount)
-	out.Messages = generateRandomlySizedMessages(msgCount, maxSize)
-	return out
+func generateRandomBlockData(txCount, evdCount, msgCount, maxSize int) (data coretypes.Data) {
+	data.Txs = generateRandomlySizedCompactShares(txCount, maxSize)
+	data.Evidence = generateIdenticalEvidence(evdCount)
+	data.Messages = generateRandomlySizedMessages(msgCount, maxSize)
+	data.OriginalSquareSize = appconsts.MaxSquareSize
```

Comment:
ideally we should calculate the square size here instead of just picking the max size, but this is going to be replaced in #692 since calculating the square size essentially requires calling `PrepareProposal`